src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -16,23 +18,28 @@
 import com.rocketden.main.model.User;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Service;
 
 import java.util.Set;
 import java.util.HashSet;
 import java.util.Random;
+import java.util.stream.Collectors;
 
 @Service
 public class RoomService {
 
     public static final int ROOM_ID_LENGTH = 6;
+    private static final Random random = new Random();
+    private static final String SOCKET_PATH = BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-user";
 
     private final RoomRepository repository;
-    private static final Random random = new Random();
+    private final SimpMessagingTemplate template;
 
     @Autowired
-    public RoomService(RoomRepository repository) {
+    public RoomService(RoomRepository repository, SimpMessagingTemplate template) {
```

Comment:
Is the `SimpMessagingTemplate template` initialized / included by default somehow?

Reply:
Yeah, it's autowired, so Spring should take care of initializing it for us and making sure a valid instance is passed to this class.